docs/03-AcceptanceTests.md

```diff
@@ -310,13 +319,65 @@ $I->seeInCurrentUrl('user/1');
 $user_id = $I->grabFromCurrentUrl('~$/user/(\d+)/~');
 ```
 
-## Selenium WebDriver
+## WebDriver: Selenium Server or PhantomJS
 
 A nice feature of Codeception is that most scenarios are similar, no matter of how they are executed.
 `PhpBrowser` was emulating browser requests but how to execute such test in a real browser like Chrome or Firefox? 
 Selenium WebDriver can drive them so in our acceptance tests we can automate scenarios we used to test manually.
 In such tests we should concentrate more on **testing the UI** than on testing functionality.
 
+"[Selenium WebDriver](http://www.seleniumhq.org/projects/webdriver/)" is the name of an API (specified by the Selenium project)
+to drive browsers automatically. Codeception supports two implementations of this API: Selenium Standalone Server and PhantomJS
```

Comment:
OK, didn't know that!  
But the solution can't be to conceal PhantomJS, since I bet there are 100 pages out there, claiming that Codeception works with PhantomJS....  
The solution is to write in the docs what you are writing here! => I'll do that. But some questions first:

1. Is there any (sort of) "official" source claiming that Phantom is dead?
1. Current version of Chrome (Windows) is 58 (Linux too?). Headless is included in 59 for Linux. Do you have an idea when this will be released? In days, weeks or months?
1. Do you have an idea when Headless for Windows will be released?